### PR TITLE
Remove mapping from xrpl asset map

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Extrinsic to remove mapping from assetIdToXRPL in xrplBridge pallet.

### CODE REVIEW

**Review by JasonTulp — COMMENTED** [review]
Instead of adding a new extrinsic, why not change the existing extrinsic to have an option for xrpl_currency?

`xrpl_currency: Option<XRPLCurrency>`

We have use this pattern a few other times in the codebase

**Comment by KarishmaBothara** [comment]
> Instead of adding a new extrinsic, why not change the existing extrinsic to have an option for xrpl_currency?
> 
> `xrpl_currency: Option<XRPLCurrency>`
> 
> We have use this pattern a few other times in the codebase

This would require migration?

File: pallet/xrpl-bridge/src/lib.rs [hunk 1]
```diff
@@ -764,14 +768,27 @@ pub mod pallet {
...
+			xrpl_currency: Option<XRPLCurrency>,
 		) -> DispatchResult {
 			ensure_root(origin)?;
-			// Validate currency to prevent errors during withdrawal
-			ensure!(xrpl_currency.symbol.is_valid(), Error::<T>::InvalidCurrencyCode);
-			<AssetIdToXRPL<T>>::insert(asset_id, xrpl_currency);
-			<XRPLToAssetId<T>>::insert(xrpl_currency, asset_id);
-			Self::deposit_event(Event::XrplAssetMapSet { asset_id, xrpl_currency });
+			// Remove mapping if xrpl_currency is none
+			if xrpl_currency.is_none() {
```
Comment (surangap) [inline pallet/xrpl-bridge/src/lib.rs:775]: nit - use the `let else` pattern to avoid the unwrap below.

**Review by surangap — COMMENTED** [review]
lgtm. added a minor comment.

**Review by JasonTulp — APPROVED** [review]

**Review by surangap — APPROVED** [review]

**Pull request merged**